CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 ## Current (in progress)
 
+### Breaking changes
+
+- Full metrics refactoring [2459](https://github.com/opendatateam/udata/pull/2459):
+  - Metric collection is now useless and will not be filled anymore.
+  - Not migration was done to remove Metrics collection in order to let users decide what to do with the remaining collection.
+  - Udata-piwik uses now InfluxDB
+  - Most of celery's tasks are removed.
+  - View's graphs are removed from admin and dashboard panel until we have accurate data to populate them.
+  - Site's metrics computation are not triggered by signals anymore.
+  - A specific celery job will be run periodically to compute site's metrics.
```

Comment:
```suggestion
  - A specific celery job needs to be periodically to compute site's metrics.
```

+ add the name of the job

And we send those data to influx too, right? we should mention it here, along w/ `udata-metrics`.